public/workbox-74d02f44.js

```diff
@@ -0,0 +1,2744 @@
+define(['exports'], (function (exports) { 'use strict';
```

Comment:
Codacy has a fix for the issue: ['use strict' is unnecessary inside of modules.](https://app.codacy.com/gh/esolidar/web3/pullRequest?prid=9467779)
```suggestion
define(['exports'], (function (exports) { 
```